### PR TITLE
add mage developer guide

### PR DESCRIPTION
**Pull request by MasterMedo**

### CODE REVIEW

File: mage/create-a-new-module.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    return mgp.Record(path=path)
+```
+
+The `get_path` is decorated with the `@mgp.read_proc` decorator, which tells Memgraph it's a `read` procedure, meaning it won't change the graph. The path is created from the `start` node, and edges are appended to it iteratively.
+
+## Running
+
+Start Memgraph and MAGE, and copy the module you developed into the `/usr/lib/memgraph/query_modules`.
+
+Instructions for a local Memgraph installation:
```
Comment (jmatak) [inline mage/create-a-new-module.md:54]: Add note that this is relevant only for Debian/Ubuntu users

File: mage/create-a-new-module.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+```
+sudo systemctl start memgraph
+cp python/random_walk.py /usr/lib/memgraph/query_modules/
+```
+
+Instructions for a docker Memgraph instance:
+```
+docker run --rm -d --name memgraph -p 7687:7687 memgraph/mage
+docker cp python/random_walk.py memgraph:/usr/lib/memgraph/query_modules/
```
Comment (jmatak) [inline mage/create-a-new-module.md:63]: In addition to this, the other possibility is having --mount, or volume set which will automatically translate local changes to the docker container storage and make work more efficient.

File: mage/create-a-new-module.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+```
+
+Then, use a Memgraph client like MemgraphLab or mgconsole to load the newly added function.
+```
+CALL mg.load('random_walk')
+```
+
+Lastly, run a query and test your module:
+```
+MERGE (start:Node {id: 0})-[:RELATION]->(mid:Node)-[:RELATION]->(end:Node)
```
Comment (jmatak) [inline mage/create-a-new-module.md:73]: Why setting an ID for the first node?

Comment (MasterMedo) [reply mage/create-a-new-module.md:73]: Should set the ids for all nodes IMO.

**Review by g-despot — APPROVED** [review]

File: mage/create-a-new-module.md (hunk 3 above)
Comment (jmatak) [reply mage/create-a-new-module.md]: Agree with you on that one, that's the main identifier inside this example.

**Pull request merged**